src/qt/votingdialog.cpp

```diff
@@ -12,7 +12,11 @@
 	#include <QtCharts/QPieSeries>
 #endif
 
-#include <QtConcurrentRun>
+#if QT_VERSION < 0x050000
+    #include <QtConcurrentRun>
+#else
+    #include <QtConcurrent/QtConcurrentRun>
```

Comment:
The m4 for the qt checks was not properly configured. This is fixed now and the #if block is no longer required.